pennylane/templates/subroutines/gqsp.py

```diff
@@ -105,8 +105,7 @@ def resource_params(self) -> dict:
         }
 
     def _flatten(self):
-        data = self.parameters
-        return data, (
+        return self.parameters, (
```

Comment:
```suggestion
        return self.data, (
```
Might as well skip an unnecessary copy.